domain/src/main/java/com/aconno/sensorics/domain/format/FormatMatcher.kt

```diff
@@ -56,15 +56,6 @@ class FormatMatcher(
         return true
     }
 
```

Comment:
Should this be removed?

Reply:
We use the new one, this one is not used anymore.